Foundation/CharacterSet.swift

```diff
@@ -486,7 +486,10 @@ extension CharacterSet : _ObjectTypeBridgeable {
     
     @_semantics("convertToObjectiveC")
     public func _bridgeToObjectiveC() -> NSCharacterSet {
-        return _wrapped
+        switch _wrapped.__wrapped {
```

Comment:
@itaiferber yes, I have 👌 seems to be working ok.

But I'm not that comfortable with the way copy-on-write is set up yet, so would like someone to double-check this please.